ricks/models.py

```diff
@@ -1,8 +1,10 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 
 class Rick(models.Model):
     """
         A model for our infinite Ricks
     """
     universe = models.CharField(max_length=255, unique=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='rick', null=True, blank=True)
```

Comment:
Cada Rick só pode estar conectado com uma conta de usuário. Ou seja, tem que ser uma relação one to one, e aqui tá one to many